tessilator/tessilator.py

```diff
@@ -1149,11 +1150,26 @@ def full_run_lc(file_in, t_target, make_plots, scc, res_table, gaia_sys=True,
     keyorder = ['run_no','id','aperture_rad','time','xcenter','ycenter',
                 'flux','flux_err','bkg','total_bkg','mag','mag_err',
                 'reg_oflux','cbv_oflux']
-    tab_format = ['%i','%s','.2f','.6f','.1f','.1f',
-                  '.6f','.6f','.6f','.6f',
-                  '.6f','.4e','.6f', '.6f']
+    tab_format = [
+        "%i",
+        "%s",
+        ".2f",
+        ".6f",
+        ".1f",
+        ".1f",
+        ".6f",
+        ".6f",
+        ".6f",
+        ".6f",
+        ".6f",
+        ".4e",
+        ".6f",
+        r".6f",
+    ]
```

Comment:
I awant to applogize for these stupid changes of ' -> " and breaking the line. I can undo it.
It's because I've set up my editor to automatically break up teh lines if they are > 80 characters and to homogenize the use f " vs '.
The benefit is that I never have to thing about formatting again - it's just done automatically when eve rI save a file. The downside is that sometimes it does stupid things like this. (Of course, there is a switch to previt it from doing it, but I'm lazy so unless it's too bad, I just let it do what it wants.)